qt/scientific_interfaces/Indirect/IndirectFitOutput.cpp

```diff
@@ -6,7 +6,7 @@
 // SPDX - License - Identifier: GPL - 3.0 +
 #include <memory>
 
-#include "IndirectFitOutputModel.h"
+#include "IndirectFitOutput.h"
```

Comment:
How does the IndirectFitOutput work in terms of MVP? It this just a rename and this still acts as a Model or does this have a new function now?

Reply:
`IndirectFitOutput` is an object for storing and accessing results from fits in the same way we use `IndirectFitData` to store input data.

Reply:
Is it worth explaining this in the Dev Docs about the Indirect File Structure?

Reply:
Is it not clear in the current dev-docs:

```The IndirectFitPlotModel also contains a pointer to the active fit function and the IndirectFitOutput from the
IndirectFittingModel, again it should only ever READ FROM THESE OBJECTS only the IndirectFittingModel should control it.```